src/development/tools/devtools/performance.md

```diff
@@ -153,8 +153,30 @@ enable the Track Layouts option:
 To see render object paint events in the timeline,
 enable the Track Paints option:
 
+
 ![Screenshot of track paints]({{site.url}}/assets/images/docs/tools/devtools/track-paints.png)
 
+## More debugging options
```

Comment:
can you add a note that users will have to reproduce activity in their app to see differences in performance

Reply:
Added a couple of paragraphs